analysis/Analysis/Section_3_1.lean

```diff
@@ -620,36 +623,63 @@ example : (5:Object) ≠ (3:Object) := by
 
 /-- Example 3.1.16 (simplified).  -/
 example : ({3, 5}:Set) ⊆ {1, 3, 5} := by
-  sorry
+  rw [SetTheory.Set.subset_def]
+  simp only [SetTheory.Set.mem_pair, SetTheory.Set.mem_triple]
+  intro x hx
+  tauto
 
 /-- Example 3.1.17 (simplified). -/
-example : ({3, 5}:Set).specify (fun x ↦ x.val ≠ 3)
- = {(5:Object)} := by
-  sorry
+example : ({3, 5}:Set).specify (fun x ↦ x.val ≠ 3) = ({5}:Set) := by
```

Comment:
Note I tweaked ` {(5:Object)}` to `({5}:Set)` here, seems like it doesn't matter and is more in line with the left side